src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/H2Translator.java

```diff
@@ -86,11 +86,7 @@ public String translate(Function f) {
         }
 
         // if it is a user-defined function
-        if (f.isUDF() && properties.isSchemaSet()) {
-            return properties.getSchema() + "." + function + "(" + expTranslated + ")";
-        } else {
-            return function + "(" + expTranslated + ")";
-        }
+        return function + "(" + expTranslated + ")";
```

Comment:
This means that the getSchema was never needed? Also, if you're gonna let this as-is, then the comment doesn't make sense anymore. The same thing applies to the other DB implementations.